frontend/elements/src/popups-sections/logout-section.ts

```diff
@@ -0,0 +1,52 @@
+import { MEDIA_UI } from '@utils/path';
```

Comment:
if `popup-sections`  is a reusable thing, it should be in `core`.

However, I don't know what `sections` means... _everything_ goes into some section, can we find a better name for what this is?

Reply:
Been moved and renamed

Reply:
Moved and renamed